docs/unified-test-documentation/dasharo-compatibility/S30-fan-speed.md

```diff
@@ -0,0 +1,107 @@
+# Dasharo Compatibility: Fan speed
+
+## Test cases common documentation
+
+**Test setup**
+
+1. Proceed with the
+   [Generic test setup: firmware](../../generic-test-setup/#firmware).
+1. Proceed with the
+   [Generic test setup: OS installer](../../generic-test-setup/#os-installer).
+1. Proceed with the
+   [Generic test setup: OS installation](../../generic-test-setup/#os-installation).
+1. Proceed with the
+   [Generic test setup: OS boot from disk](../../generic-test-setup/#os-boot-from-disk).
+
+## FAN001.001 CPU fan
+
+**Test description**
+
+This test aims to verify that the CPU fan RPM value is available to read.
```

Comment:
We should change the description of the test case. We don't check in this case is the fan speed is available to read - we check if the fan works.

Reply:
Updated